themehrdad/NetTelebot
# Added type and method, refact. Added test.

* Added Venue type (as VenueInfo). 
* Added SendVenue method.
* Added SendVenueTest() to TelegramBotClientTest. 
* Added test NullReferenceException for Venue type. 
* Added init new instance for class (in constructor) in MessageInfo.GetNewMessageInfo() (except NullReferenceException)
* Added MessageInfoVenueTest in MessageInfoParserTest
* Added type IntegerResult.
* Remove unused constructor from BooleanResult. 
* Added GetChatMembersCount method with tests.
* Added IntegerResultObject. Added test IntegerResultParserTest.
* Refact MockServer
* Change in method StopCheckUpdates(). Timer dispose and destroy. 
* Added check token on null for update method in TelegramBotClient.
